javascript/src/game.js

```diff
@@ -1,171 +1,173 @@
-const Game = function () {
-  var players          = new Array();
-  var places           = new Array(6);
-  var purses           = new Array(6);
-  var inPenaltyBox     = new Array(6);
+const Game = function() {
+  this.logs = [];
+  this.log = text => {
```

Comment:
I wanted to take this one step further to make the `console.log` call optional so tests wouldn't output a lot of garbage, but this still feels like a step in the right direction.